README.md

```diff
@@ -102,9 +102,9 @@ id="sanctuary-types-return">[1](#sanctuary-types)</sup>
     - In `a ~> a -> a` (see squiggly arrow above), `a` can be of any type.
       `Semigroup a => a ~> a -> a` adds a constraint such that the type `a`
       must now satisfy the `Semigroup` typeclass. To satisfy a typeclass means
-      to implement any function/method defined by that typeclass. In
-      addition, any implementation of such function/method must also satisfy
```

Comment:
That's the old text :) new text is:

_... any implementation of such functions/methods_